File: Laravel/app/Models/CartModel.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class CartModel extends Model
{
    use HasFactory;
    protected $table = 'tb_cart';
 
    protected $fillable = [ 
        'id_order',
        'id_produk',
        'qty',
        'harga' 
    ];
}

File: Android/app/src/main/java/com/appfranchisor/app/ui/franchisee/model/CartModel.kt
package com.appfranchisor.app.ui.franchisee.model

import com.google.gson.annotations.SerializedName

data class CartModel(
    @SerializedName("id_produk" ) var idProduk : Int?    = null,
    @SerializedName("nama"      ) var nama     : String? = null,
    @SerializedName("gambar"    ) var gambar   : String? = null,
    @SerializedName("qty"       ) var qty      : Int?    = null,
    @SerializedName("harga"     ) var harga    : Int?    = null)
